src/types.ts

```diff
@@ -462,8 +462,8 @@ export interface Transaction {
  */
 export interface OpenSeaFungibleToken extends Token {
   imageUrl?: string
-  ethPrice?: string
-  usdPrice?: string
+  ethPrice?: BigNumber
```

Comment:
what about the corresponding `utils` deserializers for these types? strings need to be (safely) turned into bignumbers using `makeBigNumber`